src/main/kotlin/uk/gov/justice/digital/hmpps/nomisvisitsmappingservice/data/MappingDto.kt

```diff
@@ -0,0 +1,27 @@
+package uk.gov.justice.digital.hmpps.nomisvisitsmappingservice.data
+
+import com.fasterxml.jackson.annotation.JsonInclude
+import io.swagger.v3.oas.annotations.media.Schema
+import javax.validation.constraints.NotBlank
+import javax.validation.constraints.NotEmpty
+import javax.validation.constraints.NotNull
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@Schema(description = "Visit creation request")
+data class MappingDto(
+
+  @Schema(description = "nomis visit id", required = true)
+  @NotNull
+  val nomisId: Long,
+
+  @Schema(description = "VSIP visit id", required = true)
+  @NotBlank
+  val vsipId: String,
+
+  @Schema(description = "Label (a timestamp for migrated ids)")
+  val label: String? = null,
+
+  @Schema(description = "Mapping type", allowableValues = ["MIGRATED", "ONLINE"], required = true)
```

Comment:
What do these different values mean?
Seems to be multiple terms here to mean the same thing - mapping type, visit type, source, visit id source?

Reply:
Good point, going with 'Mapping type'